apispec-model/src/main/scala/sttp/apispec/Schema.scala

```diff
@@ -2,14 +2,33 @@ package sttp.apispec
 
 import scala.collection.immutable.ListMap
 
-// todo: discriminator, xml, json-schema properties
+sealed trait SchemaLike
```

Comment:
Thanks, @hamnis, Probably the last thing - could you please add a comment explaining why entities `SchemaLike` and `AnySchema` were introduced? At first glance, it is not obvious why they are there, so it would have great value for future readers (and it would give me a chance to confirm, that I understand this construct / its rationale correctly :-D).